README.rst

```diff
@@ -299,6 +299,12 @@ the token in the ``Authorization`` header:
     # send bearer token
     http GET localhost:8000/api/v1/firmware/build/ "Authorization: Bearer $TOKEN"
 
+User permissions
+---------------
```

Comment:
the length of this must be equal to the length of the heading.
I would also move this section after the organization membership helpers and call it `User permissions helper` for consistency.